library/std/build.rs

```diff
@@ -100,6 +100,8 @@ fn main() {
         ("x86" | "x86_64", _) => false,
         // Missing `__gnu_h2f_ieee` and `__gnu_f2h_ieee`
         ("powerpc" | "powerpc64" | "powerpc64le", _) => false,
+        // Missing `__gnu_h2f_ieee` and `__gnu_f2h_ieee`
+        ("mips" | "mipsel" | "mips64" | "mips64le", "linux") => false,
```

Comment:
```suggestion
        ("mips" | "mipsel" | "mips64" | "mips64le", _) => false,
```

To match the above suggestion

Reply:
Done!